s4e-web/cypress/support/commands.js

```diff
@@ -23,3 +23,18 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+
+
+// TODO: Remove after Cypress upgrade to version above 5.x.y
+/**
+ * UnfetchJS library support data (XML) fetching incorrectly working in the cypress 4.x.y 
```

Comment:
Update the cypress version in comment to 6.x.y

Reply:
done